server/src/test/java/org/elasticsearch/index/query/RangeQueryBuilderTests.java

```diff
@@ -166,7 +176,7 @@ protected void doAssertLuceneQuery(RangeQueryBuilder queryBuilder, Query query,
             if (mappedFieldType instanceof DateFieldMapper.DateFieldType) {
                 fromInMillis = queryBuilder.from() == null ? null :
                     ((DateFieldMapper.DateFieldType) mappedFieldType).parseToMilliseconds(queryBuilder.from(),
-                        queryBuilder.includeLower(),
+                        !queryBuilder.includeLower(),
```

Comment:
I am not sure why this has changed?

Reply:
interesting, that slipped.. I need to investigate what has changed here